descriptions/process_state_change_type.md

```diff
@@ -0,0 +1,33 @@
+This enumeration defines types of operations that can be performed on process state objects.
+
+# Applicable to
+ - `NtChangeProcessState`
+
+# Members
+
+## ProcessStateChangeSuspend (0)
+Suspends (freezes) the associated process.
+
+|                 | Change
+| --------------- | ------
+| Extra buffer    | void (zero size)
+| Required access | `PROCESS_SUSPEND_RESUME`
+
+### Remarks
```

Comment:
Did you mean to add remarks here? If not, I guess we it can be removed.

Reply:
I guess I had a thought and then lost it =)
You can remove it; or I can do that since I'm also preparing another batch of changes for later today/tomorrow.